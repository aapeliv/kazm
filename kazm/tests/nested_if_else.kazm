int main() {
  if (true) {
  } else {
    if (false) {
    }
  }
}
