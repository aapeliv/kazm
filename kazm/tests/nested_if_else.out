void main() {
  if (true) {
  } else {
    if (false) {
    }
  }
  return;
}
